make/Images.gmk

```diff
@@ -155,7 +205,7 @@ define CreateCDSArchive
   $$(eval $$(call SetupExecute, $1_$2_gen_cds_archive_jdk, \
       WARN := Creating CDS$$($1_$2_DUMP_TYPE) archive for jdk image for $1, \
       INFO := Using CDS flags for $1: $$($1_$2_CDS_DUMP_FLAGS), \
-      DEPS := $$(jlink_jdk), \
+      DEPS := $$(FINAL_JDK_JLINK), \
```

Comment:
Does this actually interfere with the cds archive creation? I would assume they output to different files and they aren't even running `java` from the same image. If not, I would skip the whole `FINAL_JDK_JLINK` and just add `$(diff_runtime_jdk)` to `JDK_TARGETS`.

Reply:
You are right. It no longer does (the old approach did, though). I'll clean it up.